tasks/packagemgmt.yml

```diff
@@ -206,7 +206,7 @@
 - name: dnf upgrade
   become: 'yes'
   dnf:
-    name:  "*"
+    name:  "*"      # noqa 403
```

Comment:
Could you fix the spaces as well so the linting action is happy?

Reply:
Hope that done correctly... I'm not so strong in these lint rules... Sorry for that